jest.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   testRegex: "/src/.*\\.test\\.ts$",
   clearMocks: true,
   collectCoverage: true,
-  coverageReporters: ["text"],
+  coverageReporters: ["text", "lcov"],
```

Comment:
Would it be possible to only enable this if `CI == true` ? That way our local systems don't get full of reports we don't look at because we look at the terminal output